files/zh-cn/web/http/headers/content-security-policy/index.html

```diff
@@ -35,37 +35,37 @@ <h3 id="获取指令：GlossaryFetch_directive_Fetch_directives">获取指令：
 
 <dl>
  <dt>{{CSP("child-src")}}</dt>
- <dd><strong>child-src：</strong>为 <a href="/zh-CN/docs/Web/API/Web_Workers_API">web workers</a> 和其他内嵌浏览器内容（例如用{{HTMLElement("frame")}}和{{HTMLElement("iframe")}}加载到页面的内容）定义合法的源地址。</dd>
+ <dd><strong>child-src</strong>：为 <a href="/zh-CN/docs/Web/API/Web_Workers_API">Web Workers</a> 和其他内嵌浏览器内容（例如用 {{HTMLElement("frame")}} 和 {{HTMLElement("iframe")}} 加载到页面的内容）定义合法的源地址。</dd>
  <dt>
  <div class="warning">
- <p>如果开发者希望管控内嵌浏览器内容和 web worker 应分别使用{{CSP("frame-src")}}和{{CSP("worker-src")}} 指令，来相对的取代 <code><strong>child-src</strong></code>。</p>
+ <p>如果开发者希望管控内嵌浏览器内容和 web worker 应分别使用 {{CSP("frame-src")}} 和 {{CSP("worker-src")}} 指令，来相对的取代 <code><strong>child-src</strong></code>。</p>
```

Comment:
`web worker` 是否也应首字母大写？

Reply:
这个不是必须的

Reply:
前面的改了，那这里也一起改一下吧。
```suggestion
 <p>如果开发者希望管控内嵌浏览器内容和 Web Worker 应分别使用 {{CSP("frame-src")}} 和 {{CSP("worker-src")}} 指令，来相对的取代 <code><strong>child-src</strong></code>。</p>
```

这个 `</strong>` 后面是否需要空格？

```suggestion
 <p>如果开发者希望管控内嵌浏览器内容和 Web Worker 应分别使用 {{CSP("frame-src")}} 和 {{CSP("worker-src")}} 指令，来相对的取代 <code><strong>child-src</strong> </code>。</p>
```